torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1263,6 +1264,59 @@ def no_sync(self) -> Generator:
                     "`True` while in the `no_sync()` context manager"
                 m._require_backward_grad_sync = old_flag
 
+    @torch.no_grad()
+    def _clip_grad_norm_(
```

Comment:
nit: _clip_grad_norm_ => clip_grad_norm_?

Reply:
Do you mean make it public/explicit? Like `_clip_grad_norm` => `clip_grad_norm_`.

Reply:
yes, to be consistent with torch.nn.utils. clip_grad_norm_

Reply:
sure.